ecl/baremetal/v2/_proxy.py

```diff
@@ -364,15 +364,19 @@ def start_server(self, server_id, boot_mode):
         server = _server.ServerAction()
         return server.start(self.session, server_id, boot_mode)
 
-    def stop_server(self, server_id):
+    def stop_server(self, server_id, type=None):
         """Stop the Baremetal Server associated with server_id.
         This request will be accepted only when the task_state is None.
 
         :param string server_id: ID for the server.
+        :param string type: Baremetal Server shutdown mode.
```

Comment:
should not copy and paste all sentence.
```suggestion
        :param string type: Baremetal Server shutdown mode.
            A valid value is HARD or SOFT.
```